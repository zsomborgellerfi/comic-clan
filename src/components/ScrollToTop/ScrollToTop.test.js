import React from "react";
import { shallow } from "enzyme";
import ScrollToTop from "./ScrollToTop";

describe("ScrollToTop", () => {
  const wrapper = shallow(<ScrollToTop />);

  it("should render ScrollToTop correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
