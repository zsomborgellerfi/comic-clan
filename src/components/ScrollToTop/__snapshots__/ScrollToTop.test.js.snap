// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollToTop should render ScrollToTop correctly 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
